backends/advanced/src/advanced_omi_backend/routers/modules/conversation_routes.py

```
@@ -165,6 +166,118 @@ async def get_conversation_waveform(
+    # Validate time range
+    if start < 0 or start >= total_duration:
+        raise HTTPException(status_code=400, detail=f"Invalid start time: {start}s (max: {total_duration}s)")
+
+    # Get audio chunks for time range
+    try:
+        wav_bytes = await reconstruct_audio_segment(
+            conversation_id=conversation_id,
+            start_time=start,
+            end_time=end
+        )
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)